scripts/installer.nsh

```diff
@@ -0,0 +1,8 @@
+# This macro fetches the currently install MM version via MSI (if any) and uninstalls it first
+!macro customInit
+  nsExec::ExecToStack "$\"C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe$\" -command $\"$$Installer = New-Object -ComObject WindowsInstaller.Installer; $$MMProduct = $$Installer.ProductsEx('', '', 7) | Where-Object -FilterScript {$$_.InstallProperty('ProductName') -eq 'Mattermost'}; if ($$MMProduct -ne $$null) {Write-Host -NoNewline $$MMProduct.ProductCode()}$\""
+  Pop $0
+  Pop $1
+  StrCmp $1 "" 0 +1
+  ExecWait '"C:\WINDOWS\system32\msiexec.exe" /x $1 /qn'
```

Comment:
It's at least theoretically possible that `C:` isn't the system drive, so these should use environment variables instead. `C:\Windows\` is `%SystemRoot%` or `%windir%`.

Reply:
Thank you for catching that :P